frontend/scss/main.scss

```diff
@@ -1,9 +1,35 @@
 // main.scss
-#input-tournament-id {
-    justify-content: end;
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Pacifico&display=swap');
+
+html, body {
+    font-family: "Poppins";
+    font-size: 15px;
+}
+a {
+    font-weight: 500;
+}
+a:hover {
+    color: #0d6efd !important;
+}
+
+.navbar {
+    background: white !important;
+    padding: 2rem 2rem;
+    min-height: 12vh;
+}
+
+.navbar .navbar-brand a {
+    padding: 1rem 0;
+    display: block;
+    text-decoration: none;
 }
 
-.divForHome {
+.navbar-brand {
+    font-family: 'Pacifico';
```

Comment:
Defining a single `font-family` value is not good practice as loading your custom font may fail. You want to define fallback fonts for the browser to use. See https://stackoverflow.com/a/6996646

For example `"Axiforma","Helvetica Neue",Helvetica,Arial,ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,sans-serif` is the font-family value on the LeanIX Engineering blog.